_posts/2023-02-13-homebrew-4.0.0.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 4.0.0
+author: MikeMcQuaid
+redirect_from: /blog/4.0.0/
+---
+
+Today, I'd like to announce Homebrew 4.0.0. The most significant change since 3.6.0 enables significantly faster Homebrew-maintained tap updates by migrating from Git-cloned taps to JSON downloads.
+
+Major changes and deprecations since 3.6.0:
+
+- Using JSON files downloaded from [formulae.brew.sh](https://formulae.brew.sh) rather than local homebrew-core and homebrew-cask taps.
+  - **Please note: this is the largest change we have made to our update process since we split Homebrew/brew and Homebrew/homebrew-core repositories. Please bear with us, there may be a few bumps**.
+  - If you had previously set `HOMEBREW_NO_AUTO_UPDATE`, `HOMEBREW_NO_INSTALL_FROM_API` or `HOMEBREW_AUTO_UPDATE_SECS` to work around bugs or annoyances: please consider unsetting these and tweaking the values based on the new behaviour.
+  - Unless you are developing formulae or casks, you can `brew untap homebrew/core` and `brew untap homebrew/cask` to save some space.
+  - [`brew update` will now be run automatically less often (every 24 hours rather than every 5 minutes)](https://github.com/Homebrew/brew/pull/14376) and these auto-updates will be much faster as they no longer need to perform the slow `git fetch` of the huge homebrew-core and homebrew-cask taps' Git repositories.
```

Comment:
```suggestion
  - [`brew update` will now be run automatically less often (every 24 hours rather than every 5 minutes)](https://github.com/Homebrew/brew/pull/14376) and these auto-updates will be much faster as they no longer need to perform the slow `git fetch` of the huge homebrew/core and homebrew/cask taps' Git repositories.
```

Reply:
Should probably add something like this to our prose style guide and/or vale checks and pick one way of doing so